src/app/models/app.model.ts

```diff
@@ -0,0 +1,34 @@
+export interface Cards {
+  type: string,
+  background_picture_path: string,
+  background_color: string
+}
+
+export interface ActivityData {
+  title: string;
+  timeframes: {
+    daily: {
+      current: number,
+      previous: number
+    }
+    weekly: {
+      current: number,
+      previous: number
+    }
+    monthly: {
+      current: number,
+      previous: number
+    }
+  }
+}
+
+export interface TitledText {
+  title: string;
+  text: string;
+}
+
+export enum activityType {
+  'daily' = 0,
+  'weekly' = 1,
+  'monthly' = 2
+}
```

Comment:
енумы как и классы пишутся с большой буквы
каждый енум должен начинаться с большой буквы.
Если суть в том, чтобы пронумеровать типы енумов, то цифры дополнительно проставлять не нужно:
```suggestion
export enum ActivityType {
  DAILY,
  WEEKLY,
  MONTHLY
}
```